createdisk.sh

```diff
@@ -106,17 +106,14 @@ cat crio-wipe.service | ${SSH} core@${VM_IP} "sudo tee -a /etc/systemd/system/cr
 
 # Preload routes controller
 ${SSH} core@${VM_IP} -- "sudo podman pull quay.io/crcont/routes-controller:${image_tag}"
-
-if [ "${ARCH}" == "aarch64" ] && [ ${BUNDLE_TYPE} != "okd" ]; then
-   # aarch64 support is mainly used on Apple M1 machines which can't run a rhel8 kernel
-   # https://access.redhat.com/solutions/6545411
-   install_rhel9_kernel ${VM_IP}
-fi
-
 cleanup_vm_image ${VM_NAME} ${VM_IP}
 
 # Only used for macOS bundle generation
 if [ -n "${SNC_GENERATE_MACOS_BUNDLE}" ]; then
+    # workaround https://github.com/crc-org/vfkit/issues/11 on macOS 12
```

Comment:
You are only telling part of the story here, the rhel 9.2 kernel *should* be working according to its versioning, and there are different issues with aarch64 and amd64 kernels. Can you add the non-working RHEL kernel versions/arch, and the version you are downgrading to either to the issue or the commit log?